leader-election-impl/src/main/java/com/palantir/paxos/PaxosStateLogImpl.java

```diff
@@ -56,18 +55,16 @@
     private static final String TMP_FILE_SUFFIX = ".tmp";
     private static final SafeLogger log = SafeLoggerFactory.get(PaxosStateLogImpl.class);
 
-    private static Predicate<File> nameIsALongPredicate() {
-        return file -> {
-            if (file == null) {
-                return false;
-            }
-            try {
-                getSeqFromFilename(file);
-                return true;
-            } catch (NumberFormatException e) {
-                return false;
-            }
-        };
+    private static boolean checkNameIsALong(File file) {
+        if (file == null) {
+            return false;
+        }
+        try {
+            getSeqFromFilename(file);
+            return true;
+        } catch (NumberFormatException e) {
+            return false;
+        }
```

Comment:
I realise the transform is mechanical, and this is correct, but this still gives me chills in a bad way 😅